src/DotNetLightning.Core/Channel/ChannelError.fs

```diff
@@ -193,6 +193,10 @@ module private ValidationHelper =
 /// Helpers to create channel error
 [<AutoOpen>]
 module internal ChannelError =
+    let feeRateMismatch (FeeRatePerKw remote, FeeRatePerKw local) =
+        (2.0 * float (remote - local) / float (remote + local))
```

Comment:
@knocte 's comment made me realize that this should actually be
```
(2.0 * ((float remote) - (float local ) / float (remote + local) ))
```

Reply:
I mean
```fs
(2.0 * (float remote) - (float origin)) /
(float (remote + local))
```

Reply:
I've fixed this.